core_backend/app/auth/dependencies.py

```diff
@@ -24,24 +27,30 @@
 
 
 USERS = {
-    "fullaccess": {
-        "password": CONTENT_FULLACCESS_PASSWORD,
+    USER1_USERNAME: {
+        "password": USER1_PASSWORD,
+        "access_level": "fullaccess",
+    },
+    USER2_USERNAME: {
+        "password": USER2_PASSWORD,
         "access_level": "fullaccess",
     },
-    "readonly": {"password": CONTENT_READONLY_PASSWORD, "access_level": "readonly"},
 }
 
 
 def auth_bearer_token(
     credentials: HTTPAuthorizationCredentials = Depends(bearer),
-) -> None:
+) -> str:
```

Comment:
Is it reasonable for this to return the token as string? This worked so I didn't look into other ways of getting the token value into the hands of the QA/Feedback endpoints to use to fetch the user_id.

Reply:
String should be fine I think. 

Also, should we just check for user here? In the future, you would extract token, lookup user and return that to the function right?